# _dir is True if wheel goes backwards

Header header
string child_frame_id
uint32 left_wheel_data
bool left_wheel_dir
uint32 right_wheel_data
bool right_wheel_dir